templates/github/request-release.markdown

```diff
@@ -0,0 +1,5 @@
+Ok @#handle#,
+
+I requested a new release for **#project#** via Slack in `#releases` channel :+1:
+
+_If you want to get notified about new releases, make sure to follow [SonataNews](https://twitter.com/sonatanews) on Twitter!_
```

Comment:
Maybe we can integrate this via the Twitter API. The account uses IFTTT at the moment.